src/connectors/cacheDb/cacheDb.connector.js

```diff
@@ -27,6 +27,34 @@ const establishConnectionToMongo = async () => {
   }
 };
 
+const connectToBeCacheDb = async () => {
```

Comment:
is this just duplicating establishConnectionToMongo?

Reply:
yep - but that method does too much - ideally you need to let each use case get its own client - then select the database/collections - gives the most flexibility going forward in light of planned work. establishConnection is too opinionated and I felt this pattern would be cleaner for you guys going forward :)